pkg/cio/io.go

```diff
@@ -32,7 +32,9 @@ import (
 
 var bufPool = sync.Pool{
 	New: func() interface{} {
-		buffer := make([]byte, 32<<10)
```

Comment:
I don’t have context about the original value. I think any convincing change should be supported by benchmark results or data.

Reply:
I  change 100 to 5000
https://github.com/ningmingxiao/containerd/blob/test_large_tty/integration/client/container_test.go#L1903-L2017
and increase time out 
https://github.com/ningmingxiao/containerd/blob/test_large_tty/Vagrantfile#L264
it almost must happen @cpuguy83